.clippy.toml

```diff
@@ -1,13 +1,21 @@
 type-complexity-threshold = 500
 disallowed-methods = [
-    # mutating environment variables in a multi-threaded context can
+    # Mutating environment variables in a multi-threaded context can
     # cause data races.
     # see https://github.com/rust-lang/rust/issues/90308 for details.
     "std::env::set_var",
     "std::env::remove_var",
 
-    # Avoid instances of https://github.com/rust-lang/rust/issues/86470
-    "std::time::Instant::duration_since",
-    "std::time::Instant::elapsed",
-    # Clippy doesn't let us ban std::time::Instant::sub, but it knows what it did.
+    # Use parking_lot instead.
+    "std::sync::Mutex",
+    "std::sync::RwLock",
```

Comment:
i don't think this works, this is the `disallowed-methods` lint, but these are type names, rather than method/function names. i believe there's a separate [`disallowed-types` lint](https://rust-lang.github.io/rust-clippy/master/#disallowed_types) for banning types.

i think this will need to go in a separate list, like:
```toml
disallowed-types = [
    # Use parking_lot instead.
    "std::sync::Mutex",
    "std::sync::RwLock",
]
disallowed-methods = [
    # ...
]
```